openai/openai.go

```diff
@@ -93,11 +98,24 @@ type Model struct {
 	OwnedBy string `json:"owned_by"`
 }
 
+type Embedding struct {
+	Object    string    `json:"object"`
+	Embedding []float64 `json:"embedding"`
+	Index     int       `json:"index"`
+}
+
 type ListCompletion struct {
 	Object string  `json:"object"`
 	Data   []Model `json:"data"`
 }
 
+type EmbeddingList struct {
+	Object string      `json:"object"`
+	Data   []Embedding `json:"data"`
+	Model  string      `json:"model"`
+	// Usage  Usage       `json:"usage,omitempty"`
```

Comment:
should we remove this?

Reply:
openai returns prompt tokens and total tokens but didn't seem like api/embeddings supports that, can remove it as long as we're cool with just omitting it

Reply:
Let's leave it out for now and document that accordingly. We can always add it to `/api/embed` later and pass it through here